core/tasks.py

```diff
@@ -238,3 +243,24 @@ def _overall_score_and_result_type(datafile):
     result_type = ScoreResult.ResultTypes.SIMPLE
     overall_score = float(datafile.readline())
     return overall_score, result_type
+
+
+def _get_mimetype(file_path):
+    mimetype = magic.from_file(file_path, mime=True)
+    # check if mimetype has something useful
+    # empty file returns inode/x-empty
+    if mimetype == 'inode/x-empty':
+        # if no mime type, it can be null(None)
+        return None
+    unzipped_file = magic.Magic(mime=True, uncompress=True)
+    unzipped_file_mimetype = unzipped_file.from_file(file_path)
```

Comment:
For clarity, I would use a different variable name that `unzipped_file`, as the object in the variable is actually a Magic class.  Perhaps `uncompressed_magic` and `uncompressed_mimetype` would be more accurate names.